libmscore/textlinebase.h

```diff
@@ -76,6 +76,10 @@ class TextLineBase : public SLine {
       M_PROPERTY(Spatium,   beginHookHeight,       setBeginHookHeight)
       M_PROPERTY(Spatium,   endHookHeight,         setEndHookHeight)
 
+      M_PROPERTY(bool,      hasBeginText,          setHasBeginText)
+      M_PROPERTY(bool,      hasContinueText,       setHasContinueText)
+      M_PROPERTY(bool,      hasEndText,            setHasEndText)
```

Comment:
I don't see a need for these new properties. Anyway, it seems like your changes to `InspectorTextLineBase::setProperty()` just duplicate code that is already there. Do you notice a difference with your new version of that function?

Reply:
You're right, those properties aren't needed, and there's a simpler way to get it fixed while not calling `inspector->element()`. See the updated code.

Reply:
But doesn't `InspectorTextLineBase::setElement()` already do that? Do you notice a difference with your changes to that function?

Reply:
Yes, before the change #296415 is not fixed, after the change it is ;-)

Reply:
Very good. You can mark this conversation as resolved.